### PR TITLE
Feature/debug rabbit

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -29,6 +29,7 @@ APP_SECRET=
 BACKUP_CODE_SALT=
 INVITATION_SALT=
 JWT_PASSPHRASE=
+LOGGING_LEVEL=error
```
Comment (beshkenadze) [inline .env.dist]: Should be removed

Comment (mg-4xxi) [reply .env.dist]: done

File: src/Consumer/SendMessageConsumer.php [hunk 2]
```diff
@@ -38,16 +38,10 @@ public function execute(AMQPMessage $msg)
         $email = $message->email;
         $options = $message->options;
         $code = $message->code;
-        $recipient = $message->recipientId;
 
         try {
             print($this->sender->getMetaData());
```
Comment (beshkenadze) [inline src/Consumer/SendMessageConsumer.php]: Should be removed

Comment (mg-4xxi) [reply src/Consumer/SendMessageConsumer.php]: done

File: .env.dist (hunk 1 above)
Comment (beshkenadze) [reply .env.dist]: I still see the unused env variable.

**Pull request merged**